src/progress.rs

```diff
@@ -4,6 +4,8 @@ use indicatif::{ProgressBar, ProgressStyle};
 use crate::api::cage::CagesClient;
 use crate::common::CliError;
 
+const MAX_SUCCESSIVE_POLLING_ERRORS: i32 = 10; // 10 attempts at 6sec intervals (1min)
```

Comment:
I'd say 30 seconds would be enough as we'd have problems if the api threw errors for 30 seconds consecutively